tests/mock_crust/drop.rs

```diff
@@ -34,7 +34,7 @@ fn drop_node(nodes: &mut Vec<TestNode>, index: usize) {
             match node.try_next_ev() {
                 Ok(Event::NodeLost(lost_name, _)) if lost_name == name => break,
                 Ok(_) => (),
-                Err(_) => panic!("Event::NodeLost({:?}) not received", name),
+                _ => panic!("Event::NodeLost({:?}) not received", name),
```

Comment:
It emitted a warning about `Err(_)` matching all error cases and that being bad practice.